docs/models.yml

```diff
@@ -119,6 +116,14 @@ organization:
     type: relation-list
     restriction_mode: B
     to: organization_tag/organization_id
+  default_theme_id:
```

Comment:
should be required

Reply:
and if only one theme is ever in use, `default` is maybe the wrong prefix, since this suggests that others can be selected, too. Maybe `used_theme_id` or something like that (but I'm not really happy with that, either). Do you (or @GabrielInTheWorld) have a suggestion?

Reply:
Rename it into `theme_id` and the reverse relation into `theme_for_organization_id`